connection.js

```diff
@@ -82,9 +85,14 @@ function checkForAttachParametersAndConnect(id, password, connection) {
  * @returns {Promise<JitsiConnection>} connection if
  * everything is ok, else error.
  */
-export function connect(id, password, roomName) {
+export async function connect(id, password, roomName) {
     const connectionConfig = Object.assign({}, config);
-    const { jwt } = APP.store.getState()['features/base/jwt'];
+    let { jwt } = APP.store.getState()['features/base/jwt'];
```

Comment:
is there a reason for not caching the getState into a constant?

Reply:
Because it's only used once in this function

Reply:
Done